data.md

```diff
@@ -109,6 +109,21 @@ Arithmetic
     rule I1 up/Int I2 => (I1 /Int I2) +Int 1 requires I1 %Int I2 =/=K 0
 ```
 
+`logNInt` returns the log base N (floored) of an integer.
+
+```k
+    syntax Int ::= log2Int(Int) [function]
+ // ------------------------------------------
+    // log2Int is the index of the most significant bit of a number
```

Comment:
Can we add the comments in the markdown text section (not as comments in the code block)?